include/dca/phys/dca_step/cluster_solver/ctaux/structs/read_write_config.hpp

```diff
@@ -24,16 +24,20 @@ namespace ctaux {
 
 template <class Parameters>
 io::Buffer& operator<<(io::Buffer& buff, const vertex_pair<Parameters>& v) {
-  return buff << v.bands << v.e_spins << v.spin_orbitals << v.r_sites << v.HS_spin << v.tau;
+  return buff << v.bands << v.e_spins << v.spin_orbitals << v.r_sites << v.delta_r << v.HS_spin
```

Comment:
So if I understand the nature of this bug correctly the loading from the config from previous block but not including the delta_r for vertex pairs resulted in a bad config.  Why did it effect non orthogonal systems lattices?